dspace-api/src/main/java/org/dspace/content/service/ItemService.java

```diff
@@ -768,6 +769,29 @@ public Iterator<Item> findByLastModifiedSince(Context context, Date last)
      */
     int countWithdrawnItems(Context context) throws SQLException;
 
+    /**
+     * finds all items for which the current user has editing rights
+     *
+     * @param context DSpace context object
+     * @param offset  page offset
+     * @param limit   page size limit
+     * @return list of items for which the current user has editing rights
+     * @throws SQLException
+     * @throws SearchServiceException
+     */
+    public List<Item> findItemsWithEdit(Context context, String q, int offset, int limit)
```

Comment:
Again, we should add the `String q` param to the JavaDocs and describe it.

Reply:
Added the doc.